finagle-core/src/main/scala/com/twitter/finagle/ssl/SslConfigurations.scala

```diff
@@ -58,6 +68,12 @@ private[ssl] object SslConfigurations {
           case Return(tms) => Some(tms)
           case Throw(ex) => throw SslConfigurationException(ex.getMessage, ex)
         }
+      case TrustCredentials.Certificates(certs) =>
```

Comment:
Undo this change please. If you would like to add this outside the context of this pull request, we can discuss it in a separate PR.